framework/doc/content/source/interfaces/SetupInterface.md

```diff
@@ -43,6 +43,37 @@ complete list of execution flags is provided by MOOSE are listed in the "registe
 
 !listing framework/src/base/MooseApp.C start=MooseApp::registerExecFlags() end=void
 
+The default value of "execute_on" is *linear* for most of MOOSE objects with few exceptions:
+
+- The auxiliary kernels have the default value of *linear* and *timestep_end*.
+- The postprocessors have default value of  *linear* and *transfer*.
+- The controls have default value of *initial* and *timestep_end*.
+- The mult-apps have default value of *timestep_begin*.
+- The user objects have default value of *timestep_end*.
+- The outputs have default value of *initial* and *timestep_end*.
+- The default value for transfers is set to be the same of the execute_on value of its corresponding sub-applications.
+
+There are several objects in framework having special default "execute_on":
```

Comment:
can keep this sentence.
Or maybe say:
"Several objects in the framework have custom or selected default "execute_on" "